### PR TITLE
Added a recommended browsers section

### PR DESCRIPTION
**Pull request by chasat-ms**

### CODE REVIEW

**Comment by PRMerger7** [comment]
@chasat-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mattbriggs — CHANGES_REQUESTED** [review]
Can you remove the locale ( /en-us/ ). The PR reviewers will ask that you remove it. The locale will be set by the client when they click the link... :)

**Comment by mattbriggs** [comment]
@chasat-ms Looks good. Just a small change. Thanks!

**Comment by sethmanheim** [comment]
@chasat-ms @mattbriggs Just a heads up, PR reviewers might ask you to move this to the private repo (preferred location for internal contributors).

File: azure-stack/operator/azure-stack-manage-portals.md [hunk 1]
```diff
@@ -86,6 +86,9 @@ If you click the help icon (question mark) in the upper-right corner of the admi
 
 If you select **Help and support** (the question mark) in the upper right corner of the administrator portal, and then select **Azure roadmap**, a new browser tab opens and takes you to the Azure roadmap. By typing **Azure Stack Hub** in the **Products** search box, you can see all Azure Stack Hub roadmap updates.
 
+### Recommended browsers
```
Comment (PatAltimore) [inline azure-stack/operator/azure-stack-manage-portals.md:89]: Same feedback as Matt. Remove /en-us from the URL to allow localization to work correctly.  I'd add a comma after recommend browsers.

**Review by PatAltimore — CHANGES_REQUESTED** [review]
Same feedback as Matt. Remove /en-us from the URL to allow localization to work correctly. I'd add a comma after recommend browsers.

**Comment by ktoliver** [comment]
@chasat-ms - @sethmanheim is right; core PMs and content developers typically work in the private repo. Let me know if you need information about doing that, for next time; I can send you email.

The link actually should be made a relative link. Validation in the private repo would request that. I'll make that edit, if I'm not stepping on any toes.

**Comment by PRMerger6** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
The relative link is to another repo, so you can't click through in GitHub preview, and here's a test link: 

https://docs.microsoft.com/en-us/azure/azure-portal/azure-portal-supported-browsers-devices#recommended-browsers

The formatting in the second URL in the table doesn't render well in GitHub, although it looks as expected on the live site. You might change it to inline code or use escaped angle brackets in a future update:

image

**Comment by ktoliver** [comment]
("You" meaning the article author. :) )

**Comment by ktoliver** [comment]
@PatAltimore Is the PR ready to merge? 

Thanks.

**Review by PatAltimore — APPROVED** [review]

**Comment by PatAltimore** [comment]
#sign-off

**Comment by ktoliver** [comment]
Thanks, all.

**Pull request merged**